### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by miripiruni**
Fix for #136

### CODE REVIEW

File: test/fixtures.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ module.exports = function(engine) {
       options = fn;
       fn = function() {};
     }
-    return bemxjst[engine].compile(fn, options || {});
+
+    return bemxjst[engine]._compile(fn, options || {});
```
Comment (zxqfox) [inline test/fixtures.js]: o_O?

Comment (miripiruni) [inline test/fixtures.js]: @zxqfox naming? or what?

Comment (zxqfox) [inline test/fixtures.js]: Ye, why change?

Comment (miripiruni) [inline test/fixtures.js]: @zxqfox old `compile()` use `generate()`. But for coverage we need to use `require`.

**Comment by zxqfox** [comment]
👍

**Comment by zxqfox** [comment]
I'd look at benchmarks. Seems good at all. Nicely done!

**Comment by miripiruni** [comment]
@zxqfox May I merge?

File: package.json [hunk 2]
```diff
@@ -45,10 +46,11 @@
   },
   "devDependencies": {
     "browserify": "^13.0.1",
+    "istanbul": "0.4.4",
```
Comment (miripiruni) [inline package.json]: Note: add ^

File: .travis.yml [hunk 3]
```diff
@@ -5,3 +5,12 @@ node_js:
   - 4
   - 5
   - 6
+
+env:
+  global:
+    - GH_REF: github.com/bem/bh.git
```
Comment (miripiruni) [inline .travis.yml]: ops

Comment (arikon) [inline .travis.yml]: Кажется, это не нужно

**Pull request merged**